master-nonlatent/master.cfg

```diff
@@ -335,8 +335,16 @@ f_windows.addStep(
 ## f_windows_msi
 
 f_windows_msi_env = {
-    "TMP": util.Interpolate("{0}\\%(prop:buildername)s\\build\\tmpdir".format('D:\\DEV\\Buildbot' if os.getenv('ENVIRON') == 'DEV' else 'D:\\Buildbot')),
-    "TEMP": util.Interpolate("{0}\\%(prop:buildername)s\\build\\tmpdir".format('D:\\DEV\\Buildbot' if os.getenv('ENVIRON') == 'DEV' else 'D:\\Buildbot')),
+    "TMP": util.Interpolate(
```

Comment:
I would keep other code refactoring away from this PR.
Please let's keep patches focused on the title / MDBF description so it's easy to follow along with what needs to be reviewed.